change/react-native-windows-2020-11-13-08-47-42-0.62-stable.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "[0.62] port nuget 5.8.0 fix to 0.62",
```

Comment:
Since this will show up in release notes, consider leaving a comment here for the user-perspective, like "Fix build errors with Visual Studio 16.8". You can actually type any message in during `yarn change` and it will be accepted.